pages/linux/kitty.md

```diff
@@ -0,0 +1,20 @@
+# kitty
+
+> A fast, feature-rich, GPU based terminal emulator.
+> More information: <https://sw.kovidgoyal.net/kitty/>.
+
+- Open the terminal with a title of `Example`:
+
+`kitty --title {{Example}}`
```

Comment:
Maybe this could be a bit more generic like in other pages. Probably also worth adding quotes to allow titles with spaces. 👍🏻
```suggestion
- Open a terminal with the specified title for the window:

`kitty --title "{{title}}"`
```